### PR TITLE
📝 ✨ 🔥 create staging branch for release

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Create a staging branch which holds functionality for the next upcoming release

## Related issue/s
- https://github.com/ITISFoundation/osparc-simcore-clients/issues/50

### CODE REVIEW

**Comment by bisgaard-itis** [comment]
image

**Review by pcrespov — APPROVED** [review]
:tada:

File: clients/python/docs/BasicTutorial_v0.6.0.ipynb [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+    "    files_api = osparc.FilesApi(api_client)\n",
+    "    input_file: osparc.File = files_api.upload_file(file=\"file_with_number.txt\")\n",
+    "\n",
+    "    solver_key: str = \"simcore/services/comp/itis/sleeper\"\n",
+    "    solver_version: str = \"2.0.2\"\n",
+    "    \n",
+    "    solvers_api = osparc.SolversApi(api_client)\n",
+    "    solver: osparc.Solver = solvers_api.get_solver_release(solver_key=solver_key, version=solver_version)\n",
+    "\n",
+    "    solver_ports = solvers_api.list_solver_ports(solver.id, solver.version)\n",
```
Comment (bisgaard-itis) [inline clients/python/docs/BasicTutorial_v0.6.0.ipynb:159]: @pcrespov all I do to document the new ports listing endpoint is to print them here in the new version of the basic tutorial. Did you have more in mind for documenting this?

**Comment by bisgaard-itis** [comment]
After discussions with @pcrespov it was decided we will control dev features enabled/disabled through an environment variable instead of having a separate `staging` branch. I therefore close this PR and delete the `staging` branch.

**Pull request closed without merging**